edocumentation-service/src/main/java/org/dcsa/edocumentation/service/ShippingInstructionService.java

```diff
@@ -56,7 +60,28 @@ public ShippingInstructionRefStatusTO createShippingInstruction(
         updatedShippingInstruction,
         savedUtilizedTransportEquipments,
         shippingInstructionTO.consignmentItems());
-    return shippingInstructionMapper.toStatusDTO(updatedShippingInstruction);
+    return shippingInstructionMapper.toStatusDTO(shippingInstruction);
+  }
+
+  @Transactional
+  public Optional<ShippingInstructionRefStatusTO> changeState(
```

Comment:
shouldn't this also be in an Unofficial ShippingInstructionService?

Reply:
Done